docs/operation/account_administration.rst

```diff
@@ -0,0 +1,88 @@
+
+Account Administration
+======================
+
+This section highlights **MetalK8s Account Administration** which covers
+changing the default username and password for some MetalK8s services.
+
+
+Administering Grafana
+*********************
+
+A fresh install of MetalK8s has a Grafana service instance with default
+credentials: ``admin`` / ``admin``. For more information on how to access
+Grafana, please refer to :ref:`this procedure <quickstart-services-grafana>`
+
+Changing Grafana username and password
+--------------------------------------
+
+To change the default username and password for Grafana on a MetalK8s cluster,
+perform the following procedures:
+
+#. Create a file named ``patch-secret.yaml`` that has the following content:
+
+   .. code-block:: yaml
+
+      stringData:
+        admin-password: <username-in-clear>
+        admin-user: <password-in-clear>
+
+#. Apply the patch file by running:
+
+   .. code-block:: shell
+
+      $ kubectl patch secrets prometheus-operator-grafana --patch "$(cat patch-secret.yml)" -n metalk8s-monitoring
```

Comment:
Should we add the kubeconfig directly in each and every command line that use kubectl ?
 `--kubeconfig=/etc/kubernetes/admin.conf`